libopenage/renderer/stages/camera/manager.cpp

```diff
@@ -21,6 +22,9 @@ CameraManager::CameraManager(const std::shared_ptr<renderer::camera::Camera> &ca
 		camera->get_view_matrix(),
 		"proj",
 		camera->get_projection_matrix());
+
+	x_bounds = std::make_pair(XMIN, XMAX);
+	z_bounds = std::make_pair(ZMIN, ZMAX);
```

Comment:
These members should be initialized in the initializer list in the constructor above.